prisma/schema.prisma

```diff
@@ -4,6 +4,7 @@
 generator client {
   provider        = "prisma-client-js"
   previewFeatures = ["interactiveTransactions", "filteredRelationCount"]
+  binaryTargets = ["native", "darwin"]
```

Comment:
@Headscracher Not sure if you wanted me to commit this or not, but this was necessary for me to get prisma working on mac. But now that you mention I should be on node 18, maybe this is change is not necessary (I was using node 16)

<img width="1163" alt="prisma darwin error" src="https://user-images.githubusercontent.com/39889198/210153752-820ff04f-5828-4576-8a00-d35cbcb0cb04.png">